extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -1076,6 +1085,29 @@ export class AmpA4A extends AMP.BaseElement {
     throw new Error('extractCreativeAndSignature not implemented!');
   }
 
+ /**
+  * Determine the desired size of the creative based on the HTTP response
+  * headers. Must be less than or equal to the original size of the ad slot
+  * along each dimension. May be overridden by network.
+  *
+  * @param {!../../../src/service/xhr-impl.FetchResponseHeaders} responseHeaders
+  * @return {?SizeInfoDef}
+  */
+  extractSize(responseHeaders) {
+    const headerValue = responseHeaders.get(CREATIVE_SIZE_HEADER);
+    if (!headerValue) {
+      return null;
+    }
+    if (!(/[0-9]+x[0-9]+/.test(headerValue))) {
```

Comment:
If you're going to do a RegExp, why not just capture the values out of the regexp and save the split?